pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1319,6 +1319,11 @@ type GatewaySpec struct {
 	// +optional
 	SSLCertificateRef string `json:"sslCertificateRef,omitempty"`
 
+	// The name of the secret that stores ca-bundle
+	// +nullable
+	// +optional
+	CaBundleRef string `json:"caBundleRef,omitempty"`
```

Comment:
Is a "caBundle" a common enough term? It's generally better to spell things out in the CR, but if ca-bundle really is common enough, at least adding a more detailed explanation in the comment would be helpful.
```suggestion
	CertificateAuthorityBundleRef string `json:"certificateAuthorityBundleRef,omitempty"`
```

Reply:
Well, imho, caBundle is very popular abbreviation. But i can change it.

Reply:
Ok let's leave it as "caBundle". How about a more descriptive comment or maybe even a link to ca-bundle docs?

Reply:
Sure, i can insert add more descriptive comment